src/main/java/nc/recipe/vanilla/CraftingRecipeHandler.java

```diff
@@ -51,8 +51,14 @@ public static void registerCraftingRecipes() {
 		addShapedOreRecipe(NCItems.reactor_door, new Object[] {"CC", "CC", "CC", 'C', new ItemStack(NCBlocks.fission_block, 1, 0)});
 		addShapedOreRecipe(NCBlocks.reactor_trapdoor, new Object[] {"CCC", "CCC", 'C', new ItemStack(NCBlocks.fission_block, 1, 0)});
 		
+		blockCompress(NCBlocks.block_depleted_californium, "ingotCalifornium252");
+		blockCompress(NCBlocks.block_depleted_plutonium, "ingotPlutonium242");
+		blockCompress(NCBlocks.block_depleted_neptunium, "ingotNeptunium237");
 		blockCompress(NCBlocks.block_depleted_thorium, "ingotThorium230");
 		blockCompress(NCBlocks.block_depleted_uranium, "ingotUranium238");
+		blockOpen(NCItems.californium, 2, NCBlocks.block_depleted_californium);
```

Comment:
I have no idea what the number in the middle of `blockOpen` does or means, and when I came up with the numbers for these new blocks I thought it was the last number in the comprising isotope name (e.g 23**8**, 23**0**). Everything else I just modified from the lines for depleted thorium and uranium.

Reply:
And I am hoping that this is all that is necessary to declare two new blocks and textures. If there is another file that also has to be modified to declare their properties, just point me in that direction.